examples/tests/Support/AcceptanceTester.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+namespace Tests\Support;
```

Comment:
there should be a space between those two lines, and in the other Actor classes as well.